.gitmodules

```diff
@@ -1,8 +1,6 @@
 [submodule "wxWidgets"]
 	path = wxWidgets
 	url = https://github.com/wxWidgets/wxWidgets
-	ignore = dirty
```

Comment:
I can tell you what it's for. Their `.gitignore` does not contain all the various Visual Studio temporary build files. So if you build it there's some additional files that git considers for staging, this makes sure that changing revisions won't be unsuccessful when there's unexpected files and that it doesn't tag the submodule as "-dirty" just for the few additional files (doesn't show the submodule in the staging area).